### PR TITLE
Update genomescope

### PR DESCRIPTION
**Pull request by mahesh-panchal**
## PR checklist

Extracts the Kmer coverage from the model summary and emit's it into a channel. This is useful for certain tools that would like to use the kmer coverage estimate as input.

Closes #XXX 

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/genescopefk/main.nf [hunk 1]
```diff
@@ -36,6 +37,8 @@ process GENESCOPEFK {
         --output . \\
         --name_prefix ${prefix}
 
+    printf -v KMERCOV "%.2f\\n" \$( grep "^kmercov" *_model.txt | cut -d" " -f2 )
```
Comment (muffato) [inline modules/nf-core/genescopefk/main.nf:40]: TIL I learned there is `printf -v` to create new environment variables. 👍🏼 

As far as I can tell, `printf -v` is a bash builtin. Are we certain this script section _always_ runs in bash ?

File: modules/nf-core/genescopefk/meta.yml [hunk 2]
```diff
@@ -54,6 +54,10 @@ output:
       type: file
       description: GeneScope histogram summary
       pattern: "*_summary.txt"
+  - kmer_cov:
+      type: string
```
Comment (muffato) [inline modules/nf-core/genescopefk/meta.yml:58]: It's always a float, isn't it ?
```suggestion
      type: float
```

Comment (ksenia-krasheninnikova) [reply modules/nf-core/genescopefk/meta.yml:58]: Yes, it is always float

Comment (mahesh-panchal) [reply modules/nf-core/genescopefk/meta.yml:58]: The groovy class is a String, but that string is always a float as that's what the format string prints.

File: modules/nf-core/genescopefk/main.nf (hunk 1 above)
Comment (mahesh-panchal) [reply modules/nf-core/genescopefk/main.nf:40]: It works in ksh and a few other shells too, but I can add the directive at the top.

**Review by muffato — APPROVED** [review]

**Pull request merged**